forge-gui/res/cardsfolder/upcoming/ravenloft_adventurer.txt

```diff
@@ -5,8 +5,9 @@ PT:3/4
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigInitiative | TriggerDescription$ When CARDNAME enters the battlefield, you take the initiative.
 SVar:TrigInitiative:DB$ TakeInitiative
 R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Battlefield | Destination$ Graveyard | ValidLKI$ Creature.OppCtrl | ReplaceWith$ Exile | Description$ If a creature an opponent controls would die, instead exile it and put a hit counter on it.
```

Comment:
i need to ask Rules Manager what is the difference between:
* Exile with counter on it
* Exile and put counter on it